src/test/java/com/benmorant/game/holycraap/model/dao/PeopleDaoTest.java

```diff
@@ -0,0 +1,94 @@
+package com.benmorant.game.holycraap.model.dao;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.benmorant.game.holycraap.model.entity.Item;
+import com.benmorant.game.holycraap.model.entity.People;
+import java.util.Collection;
+import java.util.Optional;
+import javax.persistence.EntityManager;
+import javax.sql.DataSource;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@SpringBootTest
+public class PeopleDaoTest {
+
+  @Autowired private PeopleDao peopleDao;
+
+  @Autowired private DataSource dataSource;
+  @Autowired private JdbcTemplate jdbcTemplate;
+  @Autowired private EntityManager entityManager;
+
+  @Test
+  void injectedComponentsAreNotNull() {
```

Comment:
For the sake of consistency call this `contextLoads`.

Reply:
OK !